CMakeLists.txt

```diff
@@ -275,12 +275,11 @@ include (${SWIG_USE_FILE})
 string(REGEX MATCH "^[0-9]+[.]" SWIG_MAJOR ${SWIG_VERSION})
 
 # Find Guile and determine which version we are using.
-# Look for guile versions in this order: 3.0 > 2.2 > 2.0
+# Look for guile versions in this order: 3.0 > 2.2.1
 
 # guile library and include dir
 pkg_check_modules (GUILE3 guile-3.0 QUIET)
-pkg_check_modules (GUILE22 guile-2.2 QUIET)
-pkg_check_modules (GUILE2 guile-2.0>=2.0.9 QUIET)
+pkg_check_modules (GUILE22 guile-2.2.1 QUIET)
```

Comment:
This is failing CI because it requires *exactly* guile-2.2.1 and all of the VMs except Ubuntu 20.04. are on 2.2.9. You want to say `(GUILE22 guile-2.2>=2.2.1 QUIET)`.